resources/dicts/events/death/general/general.json

```diff
@@ -300,5 +300,28 @@
         "cat_skill": null,
         "other_cat_trait": null,
         "other_cat_skill": null
+    },
+    {
+        "tags": ["other_cat", "Newleaf", "Greenleaf", "Leaf-fall", "Leaf-bare"],
+        "camp": "Any",
+        "death_text": "m_c has always wondered where {PRONOUN/m_c/subject} came from. {PRONOUN/m_c/subject/CAP} {VERB/sneak/sneaks} out of camp, intending to find loners who might have some clues about {PRONOUN/m_c/poss} origins, when suddenly {PRONOUN/m_c/subject} {VERB/is/are} ambushed and killed by a badger. c_n finds {PRONOUN/m_c/poss} body after a long search and wonders why {PRONOUN/m_c/subject} decided to leave the safety of the c_n.",
```

Comment:
I do agree with the comment that was made in the thread that the badger feels out of place.  since the event spends so much time dwelling on the conflict of m_c not knowing where they came from, it feels weird to suddenly switch to a second important event.  They're fighting for attention too much, which isn't ideal for a short event.  I would suggest removing the specific death reason (i.e. badger) and instead just have m_c's body be found after the search.  perhaps something like "When it's realized that m_c hasn't been seen all day, a search party is sent out.  Only to find the still, cold body of m_c.  c_n is left to wonder what killed m_c and why they left camp in the first place."